examples/README.md

```diff
@@ -0,0 +1,18 @@
+<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
```

Comment:
We should try to fix the root cause of this issue instead of adding a README in the examples folder. Anyone with no knowledge behind this empty README can delete it causing the same issue again.

Reply:
removing